src/importers/lastpassCsvImporter.ts

```diff
@@ -170,7 +170,29 @@ export class LastPassCsvImporter extends BaseImporter implements Importer {
                         'Number': 'number',
                         'Name on Card': 'cardholderName',
                         'Security Code': 'code',
+                        // LP provides date in a format like 'June,2020'
+                        // Store in expMonth, then parse and modify
+                        'Expiration Date': 'expMonth',
                     });
+
+                    const exp = mappedData[0].expMonth;
```

Comment:
not too familiar with the codebase. can we rely on `mappedData[0]` to exist in this context?

Reply:
looks like yes